staging/src/k8s.io/kubectl/pkg/cmd/debug/profiles.go

```diff
@@ -173,17 +173,15 @@ func (p *restrictedProfile) Apply(pod *corev1.Pod, containerName string, target
 		return fmt.Errorf("restricted profile: %s", err)
 	}
 
+	clearSecurityContext(pod, containerName)
```

Comment:
Sorry, was this already existed in the first version of PR?. Because now we are clearing security context not only for nodes, but also podCopy and ephemeral?. I thought that we are trying to no clear security context?

Reply:
We need to first clear the security context & and then set running as non-root & drop all capabilities based on the [KEP](https://github.com/kubernetes/enhancements/tree/master/keps/sig-cli/1441-kubectl-debug#debugging-profiles).

![image](https://user-images.githubusercontent.com/18542826/219385898-ec750e5e-2cf1-4fa4-9bae-59061bebe33c.png)